### DISCUSSION TITLE
Toggle not working as expected

### DISCUSSION THREAD

**Discussion by joshuavachon25** [saadeghi/daisyui#1068] [Q&A] [answered]
When using toggle, it disappear or the background become gray (example: https://imgur.com/a/VNPH48x )
Also, the animation isn't working. I have similar problem with checkboxes and radio button. 

Here is a code example: 
`<div class="flex flex-row place-items-start p-6 gap-6">
            <p class="text-sm font-medium text-gris-f mt-2 mx-4 w-1/4 w-max-1/3">Label</p>
            <div class="flex flex-col gap-2 form-control">
                <label class=" flex flex-row-reverse  cursor-pointer gap-4">
                    <span class="text-sm font-medium text-gris-f">Label</span>
                    <input type="checkbox" class="toggle toggle-primary toggle-md" checked/>
                </label>
            </div>
        </div>`

Would anyone know why it's doing that?

**Comment by saadeghi** [comment, marked as answer]
See https://github.com/saadeghi/daisyui/issues/638
